main.py

```diff
@@ -201,9 +214,18 @@ def list_policies():
     """
     policies_list = []
     with client.context():
-        keys = PolicyModel.query().fetch(keys_only=True)
-        for key in keys:
-            policies_list.append(key.id())
+        verbose = request.args.get("verbose") == "true"
+        if (verbose):
+            policies = PolicyModel.query().fetch(keys_only=False)
+            for policy in policies:
+                policies_list.append(
+                    {"name": policy.Name, "displayName": policy.DisplayName}
+                )
+        else:
+            keys = PolicyModel.query().fetch(keys_only=True)
```

Comment:
`keys_only` has a default of `None`. If we don't want it true, we can just omit it entirely.